packages/experiments/src/components/Chiclet/Chiclet.test.tsx

```diff
@@ -2,11 +2,13 @@ import * as React from 'react';
 import * as renderer from 'react-test-renderer';
 
 import { ChicletXsmall } from './ChicletXsmall';
+import { ChicletCard } from './ChicletCard';
 import { IChicletCardProps } from './ChicletCard.types';
 
 describe('Chiclet', () => {
-  it('renders chiclet with title, icon, onClick, and url', () => {
+  it('renders Xsmall chiclet with title, icon, onClick, and url', () => {
     const chicletCardProps: IChicletCardProps = {
+      url: 'contuso.sharepoint.com',
```

Comment:
nit: it doesn't really matter what you put here, but usually we use 'contoso' :)